drizzle.config.ts

```diff
@@ -6,9 +6,9 @@ export const postgresConnectionURL = `postgres://${process.env.DB_NAME}:${proces
 export default {
   schema: "./src/db/schema",
   out: "./drizzle",
-  driver: "pg", // 'pg' | 'mysql2' | 'better-sqlite' | 'libsql' | 'turso'
+  driver: "better-sqlite", // 'pg' | 'mysql2' | 'better-sqlite' | 'libsql' | 'turso'
```

Comment:
Not sure if this driver config is used anywhere, but despite the fact that we're using bun:sqlite instead of better-sqlite3, there's no option for bun:sqlite as far as I could tell so leaving this as is since everything seems to work.